tests/core/policies/test_ted_policy.py

```diff
@@ -130,6 +136,34 @@ def test_normalization(
 
         mock.normalize.assert_called_once()
 
+    def test_label_data_assembly(
+        self, trained_policy: TEDPolicy, default_domain: Domain
+    ):
+        interpreter = RegexInterpreter()
+        encoded_all_labels = trained_policy.featurizer.state_featurizer.encode_all_labels(
+            default_domain, interpreter
+        )
+
+        attribute_data, _ = model_data_utils.convert_to_data_format(encoded_all_labels)
+        assembled_label_data = trained_policy._assemble_label_data(
+            attribute_data, default_domain
+        )
+        assembled_label_data_signature = assembled_label_data.get_signature()
+
+        assert list(assembled_label_data_signature.keys()) == [
+            f"{LABEL}_{ACTION_NAME}",
+            f"{LABEL}",
```

Comment:
Again, I thought based on the docstring we expect `label_action_text` as well?

Reply:
My reply on the docstring comment should help clarify this further.